packages/picasso/src/QuillEditor/hooks/useSubscribeToQuillEvents/test.ts

```diff
@@ -7,13 +7,19 @@ describe('useSubscribeToQuillEvents', () => {
   it('subscribes to events correctly', () => {
     const onTextChange = jest.fn
     const onSelectionChange = jest.fn
+    const onTextLengthChange = jest.fn
     const quill = ({
       on: jest.fn(),
       off: jest.fn()
     } as unknown) as Quill
 
     const { unmount } = renderHook(() =>
-      useSubscribeToQuillEvents({ quill, onTextChange, onSelectionChange })
+      useSubscribeToQuillEvents({
+        quill,
+        onTextChange,
+        onTextLengthChange,
+        onSelectionChange
+      })
     )
 
     expect(quill.on).toHaveBeenCalledTimes(3)
```

Comment:
since we added one more subscription, this test must be failing 🤔 
```suggestion
    expect(quill.on).toHaveBeenCalledTimes(4)
```

Reply:
Because we are not merging into master, ci does not run all jobs. You need to run tests locally ;-)